stl/inc/xmemory

```diff
@@ -781,6 +781,33 @@ struct _Simple_types { // wraps types from allocators with simple addressing for
     using const_pointer   = const value_type*;
 };
 
+#if _HAS_CXX23
+// STRUCT TEMPLATE allocation_result
+template <class _Ptr>

```

Comment:
I am slightly grumpy that this is not constrained by anything.

That said it is not in the paper 😞

Reply:
IMO it's good because a constraint would only complicate things. The "associated constraints are not satisfied" error is already my most hated one.

Reply:
I feel that, although other compilers actually generate good error messages for concepts